src/Playlist.js

```diff
@@ -645,7 +686,7 @@ export default class {
         let activeTrack = this.getActiveTrack();
         let trackElements = this.tracks.map((track) => {
             return track.render(this.getTrackRenderData({
-                "isActive": (activeTrack === track) ? true : false,
+                "isActive": (this.getTimeSelection().start!=this.getTimeSelection().end)?((activeTrack === track) ? true : false):true,
```

Comment:
Why not rewrite the setTimeSelection function like this : 
```
setTimeSelection(start, end) {
        this.timeSelection = {
            start,
            end,
            isChunk : (start!=end)
        };
        this.cursor = start;
    }
```
Then in the condition we call : `this.getTimeSelection().isChunk`